Purchases/Public/RCPurchases.m

```diff
@@ -722,9 +736,9 @@ - (void)offeringsWithCompletionBlock:(RCReceiveOfferingsBlock)completion
     if (self.deviceCache.cachedOfferings) {
         RCDebugLog(@"Vending offerings from cache");
         CALL_IF_SET(completion, self.deviceCache.cachedOfferings, nil);
-        if ([self.deviceCache isCacheStale]) {
+        if (self.deviceCache.isOfferingsCacheStale) {
             RCDebugLog(@"Cache is stale, updating caches");
-            [self updateCaches];
+            [self updateAllCaches];
```

Comment:
Shouldn't we just update the offerings cache?

Reply:
great catch